recipes/grass-session/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "grass-session" %}
+{% set version = "0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7155314535790145da8e2e31b0d20cd2be91477d54083a738b5c319164e7f03b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - grass_session
+
+about:
+  home: "https://github.com/zarch/grass-session"
+  license: "GPL-3.0-or-later"
+  license_family: GPL3
+  license_file: LICENSE.txt
```

Comment:
Please add a comment above the `license_file` line with a link to the following PR (https://github.com/zarch/grass-session/pull/19) as a reminder to check to see if the license file is included in the source distribution of future releases of the package (so the manually packaged copy can be removed).